CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed
 
+- The default value of `--api-base-url` is changed to
```

Comment:
Weird line break ? Is it normal?

Reply:
text wrapping at 90 characters, I believe

Reply:
I won't change the rendered output